spec/features/preassembly_run/book_using_file_manifest_spec.rb

```diff
@@ -12,7 +12,7 @@
   let(:object_staging_dir) { Rails.root.join(Settings.assembly_staging_dir, 'bb', '000', 'kk', '0000', bare_druid) }
   let(:dro_access) { { view: 'world' } }
   let(:item) do
-    Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.image).new(access: dro_access)
+    Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.book, reading_order: 'right-to-left').new(access: dro_access)
```

Comment:
To avoid needing to cut a cocina-models release just for testing purposes, we could use the same approach used for patching in `dro_access` to patch in the viewing direction?

```suggestion
    Cocina::RSpec::Factories.build(:dro, type: Cocina::Models::ObjectType.book).new(access: { view: 'world' }, structural: { hasMemberOrders: [{ viewingDirection: 'right-to-left' }] })
```